src/components/ChatApp/Settings/Settings.react.js

```diff
@@ -1375,6 +1383,7 @@ class Settings extends Component {
           </RadioButtonGroup>
           <RaisedButton
             label={<Translate text="Edit theme" />}
+            disabled={!somethingToEdit}
```

Comment:
This can be done in a one-liner I think

`disabled={this.state.theme !== 'custom'}`

Reply:
Yes. I didn't realize this. I'll do it.